.github/workflows/merge-to-stage.js

```diff
@@ -177,10 +163,6 @@ const getStageToMainPR = () => github.rest.pulls
   .then(({ data } = {}) => data.find(({ title } = {}) => title === PR_TITLE))
   .then((pr) => pr && addLabels({ pr, github, owner, repo }))
   .then((pr) => pr && addFiles({ pr, github, owner, repo }))
-  .then((pr) => {
-    pr?.files.forEach((file) => (SEEN[file] = true));
-    return pr;
```

Comment:
The method name is `getStageToMainPR` and it used to have a `return pr;` statement, would we still need that? 🤔

Reply:
I wondered the same, but it's already returned from the promise before
`.then((pr) => pr && addFiles({ pr, github, owner, repo }))` .. `addFiles` returns the PR already (well a promise that returns the PR)

Reply:
And would it hurt to explicitly have it here as well? If we change the `addFiles` method to not return a PR for whatever reason, this will fail; being in a different file, it might be difficult to spot. Since this isn't customer-facing code, I'd still vote we live with the extra line and make things explicit for any future updates.

Reply:
Fair enough 😀